test/init_local_test.go

```diff
@@ -41,6 +41,30 @@ func (s *InitLocalTestSuite) TestWithInvalidWorkdir() {
 	)
 }
 
+func (s *InitLocalTestSuite) TestListComposeFiles() {
+	require := s.Require()
+	r := s.RunCommand("compose", "list")
+	stdOut := r.Stdout()
+	require.Equal(fmt.Sprintf("[0]  local\n"), stdOut)
+}
+
+func (s *InitLocalTestSuite) TestSetComposeFile() {
+	require := s.Require()
+	r_set := s.RunCommand("compose", "set", "0")
+	require.Equal(fmt.Sprintf("Active docker compose file was changed successfully.\n"+
```

Comment:
these tests fill fail on windows most probably

Reply:
I will solve it with contains method of string library, what do you think about the tests location?

Reply:
taking into account that integration tests (only init actually) are slow, putting the tests here is alright for me.